docs/user-guide/how_to_use_mpi_plugin.md

```diff
@@ -0,0 +1,76 @@
+# MPI plugin User Guide
+
+## Introduction
+
+### Parameters of the mpi plugin
+
+```
+--master  Name of MPI master, default is "master"
```

Comment:
Let's make a list of the arguments and give examples about how to make use of them.